.eslintrc.yml

```diff
@@ -1,11 +1,17 @@
-extends: eslint:recommended
+extends:
+  - eslint:recommended
 
 env:
-  node: true
-  es6: true
   browser: true
+  es6: true
+  node: true
+
+root: true
 
 rules:
   comma-dangle: off
+  indent: [error, 2]
   no-console: off
   no-unused-vars: [error, { vars: all, args: none }]
+  quotes: [error, double]
+  semi: [error, never]
```

Comment:
these need to go to https://github.com/gregglind/shield-studies-addon-template as well, if you want to bug / think up there!  (That's the upstream!)